code/create_app.py

```diff
@@ -180,45 +187,51 @@ def conversation_with_data(request: Request, env_helper: EnvHelper):
         }
 
         return response_obj
-    else:
-        return Response(
-            stream_with_data(response),
-            mimetype="application/json-lines",
-        )
+
```

Comment:
https://pylint.readthedocs.io/en/latest/user_guide/messages/refactor/no-else-return.html

Reply:
ahaa.. this is so nice. My pet peeve is when else block literally has only 1 line that returns.